content/en/docs/concepts/security/pod-security-standards.md

```diff
@@ -187,7 +187,7 @@ fail validation.
 		<tr>
 			<td style="white-space: nowrap">SELinux</td>
 			<td>
-				<p>Setting the SELinux type is restricted, and setting a custom SELinux user or role option is forbidden.</p>
+				<p>Setting the SELinux type is restricted, and setting a custom SELinux user or role option is forbidden. This is a Linux specific field and should not be used for Windows Pods.</p>
```

Comment:
we would not add OS-specific docs to this check... no matter the OS of the pod, this policy should be enforced (since it permits omitting the field)

Reply:
So, if the field permits undefined values, we should let the pod security validation pas? I thought we can outright reject pods if they have mismatched fields in the pod or container security context